### PR TITLE
 [doc] PR for Document Flyway under Ecosystem Integrations (Doc9357) 

### PR DESCRIPTION
**Pull request by lizayugabyte**
Preview: https://deploy-preview-9701--infallible-bardeen-164bc9.netlify.app/latest/integrations/flyway/

### CODE REVIEW

File: docs/content/latest/integrations/flyway.md [hunk 1]
```diff
@@ -0,0 +1,118 @@
...
+  public void migrate(Context context) throws Exception {
+    try (Statement alter = context.getConnection().createStatement()) {
+      alter.execute("ALTER TABLE motorcycle_manufacturers 
+                    ADD COLUMN hq_address VARCHAR(50)");
+    }                   
+  }
+}
+```
+
+To check the state of the database and run the migration, execute the following commands:
```
Comment (ashetkar) [inline docs/content/latest/integrations/flyway.md:105]: Can we add few lines like below here?
"Compile the class and place its jar in the `<FLYWAY_INSTALL_DIR>/jars` directory.
It is the default location where Flyway looks for Java migration jars.
Alternatively, one can specify the jar path in the flyway configuration file using the property `flyway.locations`."

File: docs/content/latest/integrations/flyway.md [hunk 2]
```diff
@@ -0,0 +1,118 @@
...
+
+```shell
+ ./flyway info
+```
+
+```shell
+ ./flyway migrate
+```
+
+
```
Comment (ashetkar) [inline docs/content/latest/integrations/flyway.md:115]: Can we add a link to flyway documentation (https://flywaydb.org/documentation/getstarted/), may be at the end?

**Review by ashetkar — APPROVED** [review]
LGTM with a few suggestions.

File: docs/content/latest/integrations/flyway.md (hunk 1 above)
Comment (lizayugabyte) [reply docs/content/latest/integrations/flyway.md:107]: Added

File: docs/content/latest/integrations/flyway.md (hunk 2 above)
Comment (lizayugabyte) [reply docs/content/latest/integrations/flyway.md:117]: Added a link to Flyway at the top

**Pull request merged**